assets/js/membraneWebRTC.ts

```diff
@@ -383,14 +429,19 @@ export class MembraneWebRTC {
       this.localTracksWithStreams.forEach(({ track, stream }) => {
         this.connection!.addTrack(track, stream);
       });
+
+      this.connection
+        .getTransceivers()
+        .forEach((trans) => (trans.direction = "sendrecv" ? "sendonly" : trans.direction));
```

Comment:
i might be reading this wrong but I think you want a double == in your ternary clause, so
`(trans.direction == "sendrecv" ? "sendonly" : trans.direction)`

Reply:
Yes it is a bug, originally it should looks like this `trans.direction = (trans.direction == "sendrecv"? "sendonly" : trans.direction`. On the second thought it can be changed here to `trans.direction = "sendonly"`, because peerConnection will only contain transceivers with tracks to send at the moment.